physutils/tests/test_physio.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
```

Comment:
The functions in the `io.py` script are not covered by any tests. Do you think it would be worth it to add some ?

Reply:
You're totally right, I overlooked it. I transferred some from peakdet.
An important takeaway is that the way history is treated will need to include the module name as well, which will need some changes.